runbooks/runbook.md

```diff
@@ -0,0 +1,113 @@
+# UPP - Splunk Event Reader
```

Comment:
I believe here should be not `UPP - Splunk Event Reader` but `UPP - Methode Image Set Mapper`.  It looks ok for all the rest.